interface-definitions/include/babel/interface.xml.i

```diff
@@ -0,0 +1,187 @@
+        <help>Time between scheduled updates</help>
+        <valueHelp>
+          <format>u32:20-655340</format>
+          <description>Milliseconds</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 20-655340"/>
+        </constraint>
+      </properties>
+      <defaultValue>20000</defaultValue>
+    </leafNode>
+    <leafNode name="rxcost">
+      <properties>
+        <help>Base receive cost for this interface</help>
+        <valueHelp>
+          <format>u32:1-65534</format>
+          <description>Base receive cost</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 1-65534"/>
+        </constraint>
+      </properties>
+    </leafNode>
+    <leafNode name="rtt-decay">
+      <properties>
+        <help>Decay factor for exponential moving average of RTT samples</help>
+        <valueHelp>
+          <format>u32:1-256</format>
+          <description>Decay factor, in units of 1/256</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 1-256"/>
+        </constraint>
+      </properties>
+      <defaultValue>42</defaultValue>
+    </leafNode>
+    <leafNode name="rtt-min">
+      <properties>
+        <help>Minimum RTT starting</help>
+        <valueHelp>
+          <format>u32:1-65535</format>
+          <description>Milliseconds</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 1-65535"/>
+        </constraint>
+      </properties>
+      <defaultValue>10</defaultValue>
+    </leafNode>
+    <leafNode name="rtt-max">
+      <properties>
+        <help>Maximum RTT starting</help>
+        <valueHelp>
+          <format>u32:1-65535</format>
+          <description>Milliseconds</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 1-65535"/>
+        </constraint>
+      </properties>
+      <defaultValue>120</defaultValue>
+    </leafNode>
+    <leafNode name="max-rtt-penalty">
+      <properties>
+        <help>Maximum additional cost due to RTT</help>
+        <valueHelp>
+          <format>u32:0-65535</format>
+          <description>Milliseconds (0 to disables the use of a RTT-based cost)</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 0-65535"/>
+        </constraint>
+      </properties>
+      <defaultValue>150</defaultValue>
+    </leafNode>
+    <leafNode name="enable-timestamps">
+      <properties>
+        <help>Enable timestamps with each Hello and IHU message in order to compute RTT values</help>
+        <valueless/>
+      </properties>
+    </leafNode>
+    <leafNode name="channel">
+      <properties>
+        <help>Channel number for diversity routing</help>
+        <completionHelp>
+          <list>&lt;1-254&gt; interfering noninterfering</list>
```

Comment:
No need to add `&lt;1-254&gt;` - it will not work to my understanding. It's good that you listed it below in valueHelp

Reply:
Removed